example/ionic-vue3/yarn.lock

```diff
@@ -1588,7 +1598,7 @@
     unplugin "1.0.1"
 
 "@sentry/capacitor@file:.yalc/@sentry/capacitor":
-  version "2.2.0"
+  version "2.3.0"
```

Comment:
Should we set this to the latest?

```suggestion
  version "2.3.1"
```

Reply:
It shouldn't impact since we don't check if the lock file from samples are dirty, and capacitor version can always change here. I'd say we can ignore it.